src/main/kotlin/io/perfometer/statistics/ScenarioSummary.kt

```diff
@@ -0,0 +1,50 @@
+package io.perfometer.statistics
+
+import java.time.Duration
+import java.time.Instant
+import kotlin.math.ceil
+
+data class SummaryData(val name: String,
+                       val requestCount: Int,
+                       val failedRequestCount: Int,
+                       val fastestTime: Duration,
+                       val averageTime: Duration,
+                       val percentile95Time: Duration,
```

Comment:
I wander if using a dedicated class for holding request latency distribution (named e.g. `RequestLatencyDistribution`_, wouldn't be a better separation of concerns here? In the class below, you'd only feed the other class constructor with the list of request stats, and summary data would have a reference to latency distribution instance.

Then adding other percentile values and separately testing the class would be easier.

Reply:
I'm not sure if I follow your suggestion properly. Do you think of just splitting SummaryData to two data classes or some bigger refactoring?

Reply:
I think, I've just found a complicated way to describe simple thing :D Spli the SummaryData to two data classes.